datalad/core/local/create.py

```diff
@@ -572,3 +554,24 @@ def _setup_annex_repo(path, initopts=None, fake_dates=False,
             'type': 'file',
             'state': 'untracked'}
     return tbrepo, add_to_git
+
+def _procedures_exists(ds, cfg_proc):
+    # Check if specified cfg_proc(s) can be discovered, storing
+    # the results so they can be used when the time comes to run
+    # the procedure. If a procedure cannot be found, raise an
+    # error to prevent creating the dataset.
```

Comment:
I think this function should get into datalad/local/run_procedure.py to be along with other procedure support functions:

```shell
❯ git grep 'def _*get.*proc'
datalad/local/run_procedure.py:def _get_proc_config(name, ds=None):
datalad/local/run_procedure.py:def _get_procedure_implementation(name='*', ds=None):
```
and should be named more inline with other function names (although yeah -- already inconsistent above IMHO) and a little more descriptive :

not also I swapped the order of args to be more inline with other 2 functions and renamed arg so body should change too

```suggestion
def _get_proc_configs(names, ds):
    """Get specifications for discovered procedures.
    
     If not a single procedure found, raises `ValueError`
     """
```